sdk/servicebus/service-bus/src/clientEntityContext.ts

```diff
@@ -86,7 +86,7 @@ export interface ClientEntityContextBase {
  */
 export interface ClientEntityContext extends ClientEntityContextBase {
   detached(error?: AmqpError | Error): Promise<void>;
-  getReceiver(name: string, sessionId?: string): MessageReceiver | MessageSession;
+  getReceiver(name: string, sessionId?: string): MessageReceiver | MessageSession | undefined;
```

Comment:
Why is this being reverted to allow undefined? 
This would require addition of is undefined checks in multiple places, correct?

Reply:
Reverted because the error `Cannot find the receiver with name ...` is not very helpful.

Moved the check to the caller of this function, so that they can provide a more meaningful error.

Please see the changes in `serviceBusMessage.ts` file

Reply:
Got it, but wouldn't it be possible to pull in the helpful error message from errors.ts util here?

Reply:
Since these errors pertain only to this file, I chose to keep the helper method here.
In contrast, all the error checks in errors.ts are used in multiple files.

Reply:
Is there a specific advantage to it?
The disadvantage I see is the errors are not all documented in one place for the SDK which seems likely useful for anyone looking to get started.

Reply:
Fair point, fixed with https://github.com/Azure/azure-sdk-for-js/pull/2284/commits/78cae5101a773526a96b1f59945fbafca0736cc5